newIDE/app/src/MainFrame/UpdaterTools.js

```diff
@@ -95,6 +95,21 @@ export const useServiceWorkerUpdateStatus = () => {
   return serviceWorkerUpdateStatus;
 };
 
+export const updateServiceWorkerToLatestVersion = async () => {
+  const { serviceWorker } = navigator;
+  if (!serviceWorker) {
+    throw new Error('Service worker not available');
+  }
+  const registration = await serviceWorker.getRegistration();
+  if (!registration) {
+    throw new Error('Service worker registration not available');
+  }
+  await registration.unregister();
+  const cacheKeys = await caches.keys();
+  await Promise.all(cacheKeys.map(key => caches.delete(key)));
+  window.location.reload();
```

Comment:
Because of this, this function is more than just a "updateServiceWorkerToLatestVersion", it's a "clearServiceWorkerAndForceReload"